docs/lib/analytics/fragments/autotrack.md

```diff
@@ -2,8 +2,8 @@ The Amplify analytics plugin records when an application opens and closes. This
 
 To view this from the Console:
 
-```
-amplify console
+```sh
```

Comment:
we try to remove the $ sign. (it's not consistent yet but new edits shouldn't have $ signs)
Reason: if you click "copy" it'll copy the dollar sign along with the code.

Reply:
that makes sense, I'll go ahead remove from all Analytics docs then